src/components/UpdateButton/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Colors from '../../constants/Colors';
+
+import {
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+  Switch
+} from '../Globals/ConfigMetric';
+
+interface ToggleUpdateProps {
+  setCanUpdate: (value : boolean) => void;
+  canUpdate: boolean;
+}
+
+const ToggleUpdate = ({ setCanUpdate, canUpdate }: ToggleUpdateProps) => {
+  return (
+    <ConfigMetricWrapper style={{
+      marginLeft: 'auto',
+      marginRight: 45,
+      width: '30%'
+    }} borderBottom='transparent' >
+      <ConfigLableContainer style={{
+        borderRightWidth: 0,
+        flex: 1 
+      }}>
+        <ConfigLabelText/>
+      </ConfigLableContainer>
+      <SwitchContainer>
+      <Switch
+          background={canUpdate ? Colors.ventOnButtonColor : Colors.ventOffButtonColor }
+          onPress={() => setCanUpdate(!canUpdate)}>
+          <ConfigLabelText size='25px'>
+            {canUpdate ? 'Save': 'Update' }
```

Comment:
I think `Edit` and `Save` are more understandable. When you want to edit the values, you touch it and the buttons become editable. And once you're happy with it, you press `Save`